app/scripts/metamask-controller.js

```diff
@@ -1947,7 +1949,13 @@ export default class MetamaskController extends EventEmitter {
    * @returns The result of the JSON-RPC request.
    */
   handleSnapRequest(args) {
-    this._trackSnapExportUsage(args.snapId, args.handler);
+    // we're not tracking at this point in flask because we eagerly fetch insights in v2
+    if (
```

Comment:
Should we feature flag this instead of checking whether we're running Flask (by checking if local snaps are allowed) at runtime?